doc/zh-cn/discovery.md

```diff
@@ -23,12 +23,12 @@
 * [**摘要**](#摘要)
 * [**如何扩展注册中心**](#如何扩展注册中心)
     * [**基本步骤**](#基本步骤)
-    * [**以 Eureka 举例**](#以-Eureka-举例)
+    * [**以 Eureka 举例**](#以-eureka-举例)
         * [**实现 eureka.lua**](#实现-eurekalua)
-        * [**Eureka 与 APISIX 之间数据转换逻辑**](#Eureka-与-APISIX-之间数据转换逻辑)
+        * [**Eureka 与 APISIX 之间数据转换逻辑**](#eureka-与-APISIX-之间数据转换逻辑)
```

Comment:
Why we need lowercase?

Reply:
The old syntax does work but most of the codebase is using lowercase anchors.  

We should have one standard and consistent way.

Reply:
good good